Formula/n/ntfy.rb

```diff
@@ -21,18 +20,8 @@ class Ntfy < Formula
 
   def install
     system "make", "cli-deps-static-sites"
-    ldflags = %W[
-      -X main.version=#{version}
-      -X main.date=#{time.strftime("%F")}
-      -X main.commit=#{Utils.git_head}
-      -s
-      -w
-    ]
-    with_env(
-      "CGO_ENABLED" => "0",
-    ) do
-      system "go", "build", *std_go_args(ldflags:), "-tags", "noserver"
-    end
+    ldflags = "-X main.version=#{version} -X main.date=#{time.iso8601} -X main.commit=#{tap.user}"
```

Comment:
TIL about using `Homebrew` as the replacement of a git commit for cases like this 👍